internal/controller/api/v1alpha1/agent_types.go

```diff
@@ -131,6 +131,7 @@ type AgentSpec struct {
 	Hostname              string            `json:"hostname,omitempty"`
 	MachineConfigPool     string            `json:"machineConfigPool,omitempty"`
 	Approved              bool              `json:"approved"`
+	InstallationDiskPath  string            `json:"installation_disk_path"`
```

Comment:
please add description comment above this line.
@avishayt should this filed be optional?